mmdet/models/dense_heads/gfl_head.py

```diff
@@ -206,6 +206,46 @@ def anchor_center(self, anchors):
         anchors_cy = (anchors[..., 3] + anchors[..., 1]) / 2
         return torch.stack([anchors_cx, anchors_cy], dim=-1)
 
+    def forward_train(self,
```

Comment:
Is there any specific reason that we need to override this method? If so, please leave a comment on the code to indicate the difference and necessity.

Reply:
I thinks it is because another `img` key should be passed into the `loss` function so that the teacher model would run an inference process.